console/src/styles/index.css

```
@@ -0,0 +1,122 @@
+
+@custom-variant dark (&:is(.dark *));
+
+@layer base {
+  * {
+    @apply border-border outline-ring/50;
+    scrollbar-width: thin;
+    scrollbar-color: var(--border) transparent;
+  }
+  html {
+    @apply overflow-x-hidden;
+    scroll-behavior: smooth;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)